config/newsroom_videos.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "image_url": "https://i.ytimg.com/vi/4t4nAmDekDA/hqdefault.jpg",
-    "title": "AACoT&#39;24 Community Meeting, Wednesday February 7th 2024",
+    "title": "AACoT'24 Community Meeting, Wednesday February 7th 2024",
```

Comment:
@sambhavgupta0705 The merged PR doesn't contain fixes for tools/cli page and this. That only covers the tools/generator page.

Reply:
I have updated my fork with latest changes. Let me know if you want me to create new PR for these remaining changes